boards/nucleo-f767zi/include/periph_conf.h

```diff
@@ -158,6 +160,42 @@ static const spi_conf_t spi_config[] = {
 #define SPI_NUMOF           (sizeof(spi_config) / sizeof(spi_config[0]))
 /** @} */
 
+#ifdef MODULE_STM32_ETH
```

Comment:
I don't think this is needed. Did you get an error with the CI because of this ?

Reply:
yes. The Definition of the struct is wrapped in a guard block. As an alternative we could remove the guard from both, the definition and the implementation, but this would add a few byte unused memory to the binary.

Reply:
Implemented as suggested